internal/configuration/validator/configuration.go

```diff
@@ -46,6 +47,18 @@ func ValidateConfiguration(configuration *schema.Configuration, validator *schem
 		configuration.TOTP = &schema.DefaultTOTPConfiguration
 	}
 
+	if configuration.Notifier == nil {
```

Comment:
I'm curious why this got moved from the bottom of the validation to near the top?